test/instances/router.lua

```diff
@@ -19,6 +19,9 @@ _G.vshard = {
 }
 _G.ivshard = _G.vshard
 
+local vshard = _G.ivshard
```

Comment:
You should be able to use the global names right away, without assigning them to a local variable. Both `vshard` and `ivshard` are already global in this file.

Reply:
Not needed anymore, but thanks anyway